src/core/models.py

```diff
@@ -145,6 +164,8 @@ def broker_username(self) -> str:
         if not self.relation or not self.relation.app:
             return ""
 
+        return self._fetch_from_secrets("broker", "broker-username")
```

Comment:
**todo:** We can remove the return underneath, right? I'm amazed it wasn't caught by `black` or `pyright`.

Reply:
Ah, forgot to remove that, thanks!